src/js/providers/default.js

```diff
@@ -63,6 +63,8 @@ const DefaultProvider = {
     getCurrentAudioTrack: noop,
     setCurrentAudioTrack: noop,
 
+    getSeekRange: noop,
```

Comment:
`getSeekRange` must be implemented in html5.

Reply:
If this method returns nothing, what will happen for code that expects a range returned?

Reply:
By default this should return `{ start: 0, end: duration } ` for backwards compatibility.

Reply:
@DanFerrer @johnBartos ^^